frontend/express/public/core/user-management/templates/data-table.html

```diff
@@ -0,0 +1,48 @@
+<div class="manage-users-table">
+  <cly-datatable-n
+    :rows="rows">
+      <template v-slot="scope">
+          <el-table-column sortable="true" prop="full_name" :label="i18n('management-users.user')">
+              <template v-slot="rowScope">
+                  <span class="text-medium" v-html="rowScope.row.full_name"></span>
```

Comment:
"full_name" is always escaped, right?

Reply:
Actually you might consider converting this to {{rowScope.row.full_name}} instead of passing it via v-html.

Reply:
validations not applied yet. this isn't final PR for this view.